README.md

```diff
@@ -41,8 +41,31 @@ To get the path to an npm package it's best to use
 `require.resolve('the-name/package.json')`, since `require.resolve('the-name')`
 may resolve to a subdirectory of the package.
 
+You can provide multiple paths:
+
+```js
+const hash = sync([
+  require.resolve('babel-core/package.json'),
+  require.resolve('babel-preset-es2015/package.json')
+])
+```
+
+An optional seed value can also be provided:
+
+```js
+const hash = sync(require.resolve('babel-core/package.json'), 'seed value')
+```
+
 Currently only a synchronous interface is available.
 
+## API
+
+### `sync(paths[, seed])`
```

Comment:
That's the way I always used to see it. It groups the comma with the optional value it acts as a separator for. It's how Node does it, and is useful when you want to represent a [more complex argument structure](https://nodejs.org/dist/latest-v5.x/docs/api/child_process.html#child_process_child_send_message_sendhandle_options_callback).

Comparing these two:

```js
fn1(a[, b[, c]]);

fn2(a[, b][, c]);
```

It's apparent that for `fn1` you must specify a value for `b` if you want to specify a value for `c`. For `fn2`, you can just do `fn2(a, c)` - and the method will use type checking to infer which construct you meant.

Your way allows a bit of ambiguity:

```js
fn2(a, [b], [c]);
```

You can't tell if arguments are determined strictly on position or if there's some type checking, arg shifting magic happening under the covers. Yes, you absolutely could develop a convention, but the slightly less pretty format leaves little doubt.